DQMOffline/Trigger/python/MssmHbbMonitor_cfi.py

```diff
@@ -29,8 +50,29 @@
     triggerobjbtag = cms.string("hltBTagCalo30x8CSVp0p92SingleWithMatching"),
```

Comment:
why don't you simply clone the above module (or a template one)
and just overwrite the needed parameters ?